.github/workflows/before_merge.yaml

```diff
@@ -115,6 +116,7 @@ jobs:
       - name: Run Model Input Variations Tests
         run: |
           set +e
+          rm -rf tests/input-variations/
```

Comment:
Let’s indicate that this folder is autogenerated so nobody changes files there. 
Something like
tests/autogen-op-tests/